### PR TITLE
KEYCLOAK-12311 Fix minor warnings with collections in packages: forms, keys, partialimport, protocol from module "services"

### PR DESCRIPTION
**Pull request by Captain1653**

### CODE REVIEW

**Review by martin-kanis — DISMISSED** [review]
@Captain1653 LGTM thank you

File: services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java [hunk 1]
```diff
@@ -140,11 +142,7 @@ public void close() {
     }
 
     private static List<String> list(String... values) {
-        List<String> s = new LinkedList<>();
-        for (String v : values) {
-            s.add(v);
-        }
-        return s;
+        return new LinkedList<>(Arrays.asList(values));
```
Comment (stianst) [inline services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java]: Can't this be further improved to just Arrays.asList(values)?

Comment (Captain1653) [reply services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java]: Original code returned LinkedList. I have tried to save semantic. Of cource I can change it, but I'm not sure about the meaning.  As you wish :)

Comment (stianst) [reply services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java]: the method returns a List, so the expectation is that it returns a list and doesn't have to be a LinkedList.

Comment (Captain1653) [reply services/src/main/java/org/keycloak/protocol/oidc/OIDCWellKnownProvider.java]: Done

**Review by stianst — APPROVED** [review]

**Pull request merged**